.github/workflows/ci.yml

```diff
@@ -148,3 +150,32 @@ jobs:
           disable_safe_directory: true
           files: lcov.info
           token: ${{ secrets.CODECOV_TOKEN }}
+
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
+    name: Create new release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            flags: --target x86_64-unknown-linux-gnu
+            asset_name: "uhyve-${{ github.ref_name }}-linux-amd64"
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: Swatinem/rust-cache@v2
```

Comment:
With the infrequency of the releases, I don't think having a cache is very useful. I'd just remove it.